src/index.html

```diff
@@ -20,7 +20,9 @@
   </head>
 
   <body class="container-fluid">
-      <script type="text/javascript" src="{% static 'omero_iviewer/bundle1.0.0.js' %}"></script>
+      <script type="text/javascript"
+              src="{% static 'omero_iviewer/bundle.js' %}">
```

Comment:
if that is internal lib, could you suffix with application version. I think client browser won't pick up newer changes when we upgrade?